packages/perseus/src/components/zoomable.tsx

```diff
@@ -261,7 +265,16 @@ class Zoomable extends React.Component<Props, State> {
             : {};
 
         // Do a fancy little slide as we fade the contents in the first time.
-        const translateOffset = visible ? "" : " translate(0, 8px)";
+        let translateOffset = "";
+        if (!disableEntranceAnimation && !visible) {
+            translateOffset = " translate(0, 8px)";
+        }
+
+        let opacity = 1;
+        if (!disableEntranceAnimation && !visible) {
+            opacity = 0;
+        }
```

Comment:
Just a nit:

```suggestion
        const shouldSlide = !disableEntranceAnimation && !visible;
        const translateOffset = shouldSlide ? " translate(0, 8px)" : "";
        const opacity = shouldSlide ? 0 : 1;
```

1. Only have one place for the condition, so it's easier to update without missing one
2. Variable name makes it more explicit what we want to happen
3. Get to use `const` instead of `let`; prevent accidental mutation